manuscripts/esquery.py

```diff
@@ -296,30 +224,22 @@ def __get_bounds(cls, start=None, end=None):
             if end:
                 end_ts = end.replace(tzinfo=timezone.utc).timestamp()
                 end_ts_ms = end_ts * 1000  # ES uses ms
-            bounds_data = ''
+
+            bounds_data = {}
             if start:
-                bounds_data = '"min": %i,' % start_ts_ms
+                bounds_data["min"] = start_ts_ms
             if end:
-                bounds_data += '"max": %i,' % end_ts_ms
-
-            bounds_data = bounds_data[:-1]  # remove last comma
-
-            bounds = """{
-                "extended_bounds": {
-                    %s
-                }
-            } """ % (bounds_data)
-            bounds = json.loads(bounds)
+                bounds_data["max"] = end_ts_ms
 
+            bounds["extended_bounds"] = bounds_data
         return bounds
 
     @classmethod
     def __get_query_agg_ts(cls, field, time_field, interval=None,
                            time_zone=None, start=None, end=None,
                            agg_type='count', offset=None):
         """
-        Create a string with an aggregation DSL query for getting the time series
-        values for field.
+        Create an aggregation DSL object for getting the time series values for field.
```

Comment:
Isn't it really a "es_dsl aggregation object" instead of a "aggregation DSL object"?

Reply:
Yeah, my bad. Thanks!